sklearn/svm/_base.py

```diff
@@ -617,6 +618,7 @@ def _validate_for_predict(self, X):
 
     @property
     def coef_(self):
+        """ndarray : Weights assigned to the features."""
```

Comment:
@glemaitre, what do you think of using something like:
```suggestion
        """Return the weights assigned to the features.
        
        ndarray : Weights assigned to the features.
        """
```

Reply:
Doesn't this have to be documented just like a method?